### PR TITLE
REF: Android file save

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/2wrkfdwdc52jvvkt1q7mehv7uw?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/filesaveandroid-1620185189.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/41xmxngay4m2jpvq7wjfkft63m?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/filesaveandroid-1620185916.apk)

File: blue_modules/fs.js [hunk 1]
```diff
@@ -26,36 +26,14 @@ const writeFileAndExport = async function (filename, contents) {
         RNFS.unlink(filePath);
       });
   } else if (Platform.OS === 'android') {
-    const granted = await PermissionsAndroid.request(PermissionsAndroid.PERMISSIONS.WRITE_EXTERNAL_STORAGE, {
-      title: loc.send.permission_storage_title,
-      message: loc.send.permission_storage_message,
-      buttonNeutral: loc.send.permission_storage_later,
```
Comment (limpbrains) [inline blue_modules/fs.js:32]: I guess these locale strings are no longer needed and can be removed from `en.json` ?

**Review by limpbrains — CHANGES_REQUESTED** [review]
Also `loc.send.permission_storage_denied_message` ?

File: blue_modules/fs.js (hunk 1 above)
Comment (limpbrains) [reply blue_modules/fs.js:32]: Also `loc.send.permission_storage_denied_message`

**Review by Overtorment — APPROVED** [review]
lgtm

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/k6n1vymzgjrnkk8xu772g756x8?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/filesaveandroid-1620222334.apk)

**Comment by Overtorment** [comment]
@marcosrdz conflicts

**Comment by marcosrdz** [comment]
> @marcosrdz conflicts

fixed

**Comment by Overtorment** [comment]
aaaaaand now all the tests failed

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/cvc5ge0bg4fdaw0tzu3hqnyu1r?device=pixel4

[download apk](http://filestorage.bluewallet.io:1488/filesaveandroid-1620742915.apk)

**Comment by Overtorment** [comment]
this was never approved by @ncoelho so it never got merged and now it has conflicts.
a good example of forgotten PR @marcosrdz

**Comment by marcosrdz** [comment]
RN Document Picker recently included folder selection for Android. I have a local branch using this approach in preference to this approach

**Pull request closed without merging**